projects/admin/src/app/service/patron.service.ts

```diff
@@ -168,6 +168,16 @@ export class PatronService {
     return this.getLoans(query, '-end_date');
   }
 
+  /**
+   * Get circulation statistics about a patron
+   * @param patronPid - string : the patron pid to search
+   * @return Observable
+   */
+  getCirculationInformations(patronPid: string): Observable<any> {
+    const url = `/api/patrons/${patronPid}/circulation_informations`;
```

Comment:
Inject `ApiService` to `PatronService` and use `[this._apiService.getEndpointByType('patrons'), patronPid, 'circulation_informations'].join('/')`